tmt/steps/provision/__init__.py

```diff
@@ -14,8 +14,9 @@
 import tmt
 import tmt.utils
 
-# Timeout in seconds of waiting for a connection
-CONNECTION_TIMEOUT = 60 * 4
+# Timeout in seconds of waiting for a connection after reboot
+# (long enough to allow operations such as system upgrade)
+CONNECTION_TIMEOUT = 60 * 60
```

Comment:
Maybe lets add a FIXME to revert this back?

Reply:
Good point, will one.